lib/Backends/NNPI/tests/NNPIParameterSweepTest.cpp

```diff
@@ -18,3 +18,91 @@
 using namespace glow;
 
 std::set<std::string> glow::backendTestBlacklist = {};
+
+struct SoftwareOnlyTests {
+  SoftwareOnlyTests() {
+    // If USE_NNPI_THRESHOLD is set, we should have empty blacklist
+    auto useNnpiThreshold = getenv("USE_NNPI_THRESHOLD");
```

Comment:
What threshold is supposed to be used if `USE_NNPI_THRESHOLD`? Is part of the PR missing?

Reply:
it's part of our internal testing, removing it